### PR TITLE
#3bdgrp

### PR DESCRIPTION
**Pull request by Yashi002**

### CODE REVIEW

File: src/views/update-password.vue [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+// TODO check why this doesn't works when the confirm value changes
+defineRule('confirmed', (value, params) => {
+  return confirmed(value, params) ? true : 'Password must match';
+});
+
+export default defineComponent({
+  name: "update-password",
+  data() {
+      return {
```
Comment (ymaheshwari1) [inline src/views/update-password.vue:110]: Please make the indentation consistent

File: src/views/update-password.vue [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      eye,
+      formData,
+    };
+  },
+});
+</script>
+
+<style scoped>
+.update-password-container {
+  width: 375px;
```
Comment (dt2patel) [inline src/views/update-password.vue:226]: change to max width

File: src/views/update-password.vue [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+}
+
+img {
+  margin-bottom: 25px;
+  padding: 16px;
+}
+
+.flex {
```
Comment (dt2patel) [inline src/views/update-password.vue:234]: unused class

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/services/UserService.ts [hunk 4]
```diff
@@ -31,10 +31,19 @@ const setUserTimeZone = async (payload: any): Promise <any>  => {
     data: payload
   });
 }
+const updatePassword = async (payload: any): Promise <any>  => {
+  return client({
```
Comment (ymaheshwari1) [inline src/services/UserService.ts:35]: Why we have used client() here and not api()?

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Comment by Yashi002** [comment]
Closed this PR, as branch can't be found

File: src/views/update-password.vue (hunk 1 above)
Comment (Yashi002) [reply src/views/update-password.vue:110]: Done

File: src/views/update-password.vue (hunk 2 above)
Comment (Yashi002) [reply src/views/update-password.vue:226]: Done

**Pull request closed without merging**